client-reactjs/.env.sample

```diff
@@ -3,4 +3,10 @@ REACT_APP_PROXY_URL=http://localhost:3000
 REACT_APP_LDAP_SEARCH_ENABLED=true
 
 # npm package version
-REACT_APP_VERSION=$npm_package_version
\ No newline at end of file
+REACT_APP_VERSION=$npm_package_version
+
+## Azure SSO
+REACT_APP_APP_AUTH_METHOD    = azure_ad
+REACT_APP_AZURE_CLIENT_ID    = 
```

Comment:
Other than the CLIENT_ID, are there any secrets used in the AD integration? We should also add this to Github secrets in 'dev'. I am assuming, we can test this with our dev environment too? Maybe, the 'dev' URL need to be added in App registration?